framework/src/postprocessors/AxisymmetricCenterlineAverageValue.C

```diff
@@ -34,7 +34,7 @@ AxisymmetricCenterlineAverageValue::AxisymmetricCenterlineAverageValue(
 Real
 AxisymmetricCenterlineAverageValue::volume()
 {
-  return _current_side_elem->volume();
+  return _current_side_volume;
```

Comment:
Whoa...this seems like a nasty bug. how much does it change the answer in some of these tests?

Reply:
Fortunately, volume is not used that often because we accumulate integrals. In the cases where DGDiffusion kernels are used the differences are not too large because volume is used for computing the penalty parameter which doesn't matter for sufficiently small elements. 

The problem where I noticed it, I wanted to compute an average temperature for cross section interpolation. I expected 800 K and got 9000 K.

Reply:
I guess `_current_side_volume` is also unaware of the fact that it's axisymmetric (because it would actually be zero since it's on the centerline). Is that right?

Reply:
No it will know that. If you look at `_coord[qp]` in `Assembly`:

```
    case Moose::COORD_RZ:
      for (unsigned int qp = 0; qp < n_points; qp++)
        coord[qp] = 2 * M_PI * q_points[qp](rz_radial_coord);
      break;
```

so it includes the r-coordinate. For `r=0`, `_coord` is zero.

Reply:
Actually reading what this object is supposed to do, a zero will be bad news here. 
You are really after the height of a line segment in the z-direction and not after the area of the side.

Reply:
I reverted this change and added a comment so we remember in the future.